packages/ppx/test/css-support/css-support.expected.re

```diff
@@ -1035,8 +1035,8 @@ CssJs.unsafe({js|colorScheme|js}, {js|light|js});
 CssJs.unsafe({js|colorScheme|js}, {js|dark|js});
 CssJs.unsafe({js|colorScheme|js}, {js|light dark|js});
 CssJs.unsafe({js|colorScheme|js}, {js|dark light|js});
-CssJs.unsafe({js|colorScheme|js}, {js|only light|js});
-CssJs.unsafe({js|colorScheme|js}, {js|light only|js});
```

Comment:
Not sure whether to remove unsupported properties of the above sort or to implement them in this PR! (the former makes sense to me)

Reply:
Does seems problems related on parsing wrongly the property (I believe keeping the component_value_in_prelude is necessary on the parsing face)